shared/elements/gaia_confirm/style.css

```diff
@@ -85,6 +88,10 @@ p strong {
   font-weight: 300;
 }
 
+.stronger {
+  font-weight: 500;
+}
```

Comment:
My preference would be to just change the above code. I don't think it should break anything. I'm also ok if you want to play it safe.